pyro/distributions/omt_mvn.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import, division, print_function
 
 import torch
-from torch.autograd import Function
+from torch.autograd import Function, variable
```

Comment:
@neerajprad is this new style in Pytorch to create variables?

Reply:
@fehiepsi - `variable` was introduced sometime back to incorporate support for scalars, e.g. using `variable(1)` will give a 0 dim tensor wrapped inside a `Variable`. In that sense, it is more general than `Variable`. I think soon Variable and Tensor APIs will be fused into a single `tensor` type.